AirCasting/CreateSessionViews/SelectPeripheralView.swift

```diff
@@ -136,7 +136,7 @@ struct SelectPeripheralView: View {
                                        userAuthenticationSession: userAuthenticationSession,
                                        sessionContext: sessionContext,
                                        peripheral: selection)
-                destination = AnyView(SyncingABView(viewModel: viewModel, baseURL: urlProvider, creatingSessionFlowContinues: $creatingSessionFlowContinues))
+                destination = AnyView(SDSyncABView(viewModel: viewModel, creatingSessionFlowContinues: $creatingSessionFlowContinues))
```

Comment:
I named it SyncingABView because it is the view which is showed when the AB is syncing. SDSyncABView seems a bit less descriptive, maybe we could find a name which is a mix of those two? :D